lib/state/appState.js

```diff
@@ -68,6 +68,7 @@ appState.getInitial = function() {
 
   return {
     dropMenu: false,
+    howToUpdateKBLink: 'https://tidepool-project.helpscoutdocs.com/article/6-how-to-install-or-upgrade-the-tidepool-uploader-gen',
```

Comment:
Do we want to have a redirecting url here in case the end url changes?  It would be easier to change the redirecting url than the wait for the uploader to be distributed.